backend/Makefile

```diff
@@ -298,6 +298,10 @@ endef
 test-deps: get-shell-check get-detekt get-mock-swiftlint get-trivy install-plugin-dependencies
 .PHONY: test-deps
 
+unit-test-no-deps:
+	$(call run-pytest,unit tests,-m "not integtest and not end2end")
+.PHONY: unit-test
```

Comment:
This should be `.PHONY: unit-test-no-deps` to match the above target.

Note also we plan to revisit/overhaul how the test dependencies are installed in the future as well, so this is a useful target to have in the meantime :)